action.yaml

```diff
@@ -21,12 +21,18 @@ inputs:
     description: Location of the files to analyze
     required: false
   calc:
-    description: Comma-separated list of metrics to be used
+    description: Comma-separated list of metrics to be used. GATE metric is supported for TiCS Viewer versions higher than 2022.2.x. If not specified, GATE will be used by default.
     required: false
     default: GATE
+  recalc:
+    description: Comma-separated list of metrics to be recalculated. GATE metric is supported for TiCS Viewer versions higher than 2022.2.x.
+    required: false
   clientData:
     description: A custom client-data token for the purpose of the Client Viewer functionality.
     required: false
+  codeType:

```

Comment:
please use camelCase everywhere. This inconsistent with other mentions of codeType.

Reply:
we will use lowercase as this is the same for the CLI